packages/fetchai/contracts/erc1155/contract.py

```diff
@@ -48,22 +48,9 @@ class Performative(Enum):
         CONTRACT_ATOMIC_SWAP_BATCH = "contract_atomic_swap_batch"
         CONTRACT_SIGN_HASH = "contract_sign_hash"
 
-    def __init__(
```

Comment:
not sure why this has to change

Reply:
subclasses don't define custom constructors anymore, but can add custom attributes in `load`